plan.tex

```diff
@@ -57,15 +70,32 @@ \subsubsection{Middleware}
 we will have to do on IDF.
 
 \paragraph{Workflow} needs to be functioning at scale for DP0.2, ideally we should basic workflow
-early on (milestone DP-MW-M-02). Then more tooling such as restarting failed jobs (DP-MW-M-03)
+early on (milestone DP-MW-M-02). Then more tooling such as restarting failed jobs (DP-MW-M-03).
+
+From the construction side we have BPS as a deliverable which may be useful on IDF also. We shall evaluate BPS as an option later in 2020 (DP-MW-M07). See \citeds{LDM-636} \citeds{LDM-633} \citeds{DMTN-123}.
```

Comment:
We prefer one sentence per line to make diffs easier.

Punctuation between the LDM mentions?

Reply:
ok